### PR TITLE
Setting up virtual environment and makefile.

### PR DESCRIPTION
**Pull request by ayuhsya**
Plis Merge. -^-

### CODE REVIEW

File: init.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+pip install virtualenv
+virtualenv -p /usr/bin/python3 venv
+activate(){
+	source venv/bin/activate
+}
+pip install -r requirements.txt
+echo "Run sample code with python3 wiki_cat.py"
```
Comment (abinashmeher999) [inline init.sh:8]: A better commit message for this change. Nice choice of filename though :+1:

**Comment by abinashmeher999** [comment]
First of all we will need a meaningful name for the PR like what does this aim to achieve.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+language: python
+install: "pip install -r requirements.txt"
+script: wiki_cat
```
Comment (abinashmeher999) [inline .travis.yml:3]: Does this mean Travis CI is fully setup. If not maybe this should wait for a different PR.

**Review by abinashmeher999 — CHANGES_REQUESTED** [review]
Seems like 2 files `Makefile` and `init.sh` are pretty redundant. Better to have just one. This PR will need a little more cleaning and then it will be good to merge :smile:

**Comment by ayuhsya** [comment]
@abinashmeher999 This also has files related to another issue. Shall I clean them?

**Comment by abinashmeher999** [comment]
@ayuhsya Yes please move them to another branch using which you can send a different PR.

**Pull request closed without merging**